lib/Chat/ChatManager.php

```diff
@@ -711,7 +711,7 @@ private function searchIsPartOfConversationNameOrAtAll(string $search, string $r
 		return false;
 	}
 
-	public function deleteExpiredMessages(int $roomId): void {
+	public function deleteExpiredMessages(?int $roomId = null): void {
 		$this->commentsManager->deleteCommentsExpiredAtObject('chat', (string) $roomId);
```

Comment:
I would say since we don't use it, don't offer it:
```suggestion
	public function deleteExpiredMessages(): void {
		$this->commentsManager->deleteCommentsExpiredAtObject('chat');
```